tests/unit/frameworks/pytorch/imagenet_util_test.py

```diff
@@ -19,26 +19,80 @@
 #
 import io
 import pickle
+# import subprocess
 import tempfile
 import unittest
+from pathlib import Path
 
 from nupic.research.frameworks.pytorch.imagenet.network_utils import create_model
 from nupic.research.frameworks.pytorch.model_compare import compare_models
-from nupic.research.frameworks.pytorch.model_utils import serialize_state_dict
+from nupic.research.frameworks.pytorch.model_utils import (
+    serialize_state_dict,
+    set_random_seed,
+)
 from nupic.research.frameworks.pytorch.models.resnets import resnet50
+from nupic.torch.modules import rezero_weights
+
+CHECKPOINTS_DIR = Path(__file__).parent / "checkpoints"
+
+TEST_MODEL_ARGS = dict(config=dict(
+    activation_params_func=lambda *_: dict(percent_on=0.5, local=True),
```

Comment:
Don't we have to set 'defaults_sparse=True'?

Reply:
I am overriding the default sparse to create smaller checkpoints, we just care about the network structure for checkpoints. The default sparse creates large checkpoints (~100M)

Reply:
👍